helix-tui/src/backend/crossterm.rs

```diff
@@ -32,10 +32,12 @@ fn vte_version() -> Option<usize> {
 }
 
 /// Describes terminal capabilities like extended underline, truecolor, etc.
-#[derive(Copy, Clone, Debug, Default)]
+#[derive(Clone, Debug, Default)]
 struct Capabilities {
     /// Support for undercurled, underdashed, etc.
     has_extended_underlines: bool,
+    /// Support for resetting the cursor style back to normal.
+    cursor_normal: Option<String>,
```

Comment:
```suggestion
    reset_cursor_command: Option<String>,
```

Reply:
This could just be a `String` and we would default to `\x1B[0 q` in `from_env_or_default`